lorawan-device/README.md

```diff
@@ -3,72 +3,32 @@
 [![Latest Version]][crates.io]
 [![Docs]][doc.rs]
 
-This is an experimental LoRaWAN device stack. It can be tested by using the
-example in [the sx12xx-rs repository](https://github.com/lthiery/sx12xx-rs). You
-may also consider the example from
-[the Drogue Device framework](https://github.com/drogue-iot/drogue-device/).
+This is an experimental LoRaWAN device stack with both non-blocking (`nb_device`) and async (`async_device`) 
+implementations. Both implementations have their respective `radio::PhyRxTx` traits that describe the radio interface
+required. 
 
-The device stack supports two modes, both designed for concurrency in non-threaded environments:
+Note: The `external-lora-phy` feature provides the `lora-phy` crate as an async implementation of `radio::PhyRxTx`. 
 
-* A state machine implementation based off [an article by Ana Hobden](https://hoverbear.org/blog/rust-state-machine-pattern/).
-* An async-await implementation that can be used with async radio interfaces.
-
-## State machine implementation
-
-There are two super-states that the Device can be in:
-
-- **NoSession**: default state upon initialization
-- **Session**: achieved after successful OTAA
-
-A state machine diagram is provided in `src/state_machines/session`
-
-The following LoRaWAN features are implemented:
+LoRaWAN 1.0.x is approximately implemented:
 
 - Class A device behavior
+- Class C device behavior (async only)
 - Over-the-Air Activation (OTAA) and Activation by Personalization (ABP)
-- Regional support for US915, and EU868
-- Supports CFList in JoinAccept for EU868
+- Regional support for AS923_1, AS923_2, AS923_3, AS923_4, AU915, EU868, EU433, IN865, US915
```

Comment:
Probably worth mentioning that region-specific power limits are not yet enforced (see issue #168)

Reply:
good idea! thanks